src/pykeen/datasets/base.py

```diff
@@ -584,7 +585,59 @@ def _load_validation(self) -> None:
         pass  # already loaded by _load()
 
 
-class SingleTabbedDataset(LazyDataset):
+class TabbedDataset(LazyDataset):
+    """This class is for when you've got a single TSV of edges and want them to get auto-split."""
+
+    ratios = (0.8, 0.1, 0.1)
+    _triples_factory: Optional[TriplesFactory]
+
+    def __init__(
+        self,
+        cache_root: Optional[str] = None,
+        eager: bool = False,
+        create_inverse_triples: bool = False,
+        random_state: RandomHint = None,
+    ):
+        """Initialize dataset.
+
+        :param cache_root:
+            An optional directory to store the extracted files. Is none is given, the default PyKEEN directory is used.
+            This is defined either by the environment variable ``PYKEEN_HOME`` or defaults to ``~/.pykeen``.
+        :param eager: Should the data be loaded eagerly? Defaults to false.
+        :param create_inverse_triples: Should inverse triples be created? Defaults to false.
+        """
+        self.cache_root = self._help_cache(cache_root)
+
+        self._triples_factory = None
+        self.random_state = random_state
+        self.create_inverse_triples = create_inverse_triples
+        self._training = None
+        self._testing = None
+        self._validation = None
+
+        if eager:
+            self._load()
+
+    def _get_path(self) -> Optional[str]:
```

Comment:
what is the purpose of this method?

Reply:
If an inheriting class has a single path, then it will get included in the triples factory so you know where the data came from

Reply:
I put a better explanation in f78d4c7b and also updated the code to use the new `metadata` tag

Reply:
👌